_rules/image-no-text-0va7u6.md

```diff
@@ -33,20 +34,21 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [rendered image resources][] in a [web page][].
+This rule applies to any [image resources][rendered image resources] rendered in a [web page][].
 
 ## Expectation
 
-For each test target, its [visible pixels][visible] do not contain text expressing anything in a [human language], except if at least one of the following is true:
+Each test target has no [visible][] [text][human language], except if at least one of the following is true:
 
-- **essential**: the [visible pixels][visible] of the test target contain text for which its presentation is [essential][]; or
-- **not significant**: the [visible pixels][visible] of the test target contain text and the text is not a significant part of the visible part of the image or
-- **decoration**: the test target belongs to an [embedded image][] that is [purely decorative][].
+- <dfn id="0va7u6:decorative">decorative</dfn>: The image with text is [purely decorative][]; or
+- <dfn id="0va7u6:incidental">incidental</dfn>: The text is not a [significant][insignificant] part of the image; or
+- <dfn id="0va7u6:essential">essential</dfn>: Ensuring consistent presentation of the text is [essential][].
```

Comment:
With the definitions here, I think that example descriptions should refer to them where relevant.
For example, Passed example 4 (Time Square):
> `This image resource referenced by the `object` element contains text, but it is not the most significant content.`
should be:
> `This image resource referenced by the `object` element contains text, but it is [incidental](#0va7u6:incidental).`